PopcornFX/README.md

```diff
@@ -0,0 +1,207 @@
+# PopcornFX Assets - O3DE Multiplayer Sample
+
+The Multiplayer Sample includes nine visual FX assets created with PopcornFX. This README documents the these FX assets so that you can customize them or deconstruct them to learn how to build PopcornFX assets of your own.
```

Comment:
Is nine relevant? I would avoid specifying actual number so we don't have to keep updating the document.

FX, while common term in game industry, should be defined on first use.

May also want to call out not all of these FX are currently used in MPS.

Reply:
Changed nine to "several"

Changed to consistently use the acronym "VFX" and define it after first use.

I'm hesitant to call out assets as not being used because there are feature requests to implement them.